drivers/gpio/gpio_mcux.c

```diff
@@ -312,10 +312,12 @@ static int gpio_mcux_manage_callback(const struct device *dev,
 
 static void gpio_mcux_port_isr(const struct device *dev)
 {
-	const struct gpio_mcux_config *config = dev->config;
+	const struct gpio_mcux_config *config;
 	struct gpio_mcux_data *data = dev->data;
 	uint32_t int_status;
 
+	config = dev->config;
```

Comment:
Instead of doing this, we could probably use `ARG_UNUSED` macro on these variables, can you try that?

Reply:
Updated, please help review again. Thanks a lot.